app/helpers/responses_helper.rb

```diff
@@ -0,0 +1,26 @@
+module ResponsesHelper
+  def display_value_for_number(response)
+    return if !response.has_number?
+    if response.has_currency?
+      "#{prefix(response)}#{response.number} #{postfix(response)}"
+    elsif response.has_percentage?
+      "#{response.number}%"
+    else
+      response.number
+    end
+  end
+
+  def prefix(response)
```

Comment:
These don't feel quite right, but I also recognize that the use of helpers here is also just a concession to us not having better options in this codebase yet, so I'd be willing to just keep this whole thing in mind for future refactoring.